_layouts/docs.html

```diff
@@ -1,7 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {% include header.html %}
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-173448502-1"></script>
+      <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-173448502-1');
```

Comment:
What google account is this tied to?

Reply:
My personal one. I was thinking about setting up an account just for Microshare so anyone can use it. I can easily swap it out.